Homework №2 from 16.09.2022/task1.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+
+unsigned long recursionFibCount(unsigned long number)
+{
+    if (number <= 0)
```

Comment:
```suggestion
    if (number == 0)
```
Сюда передают unsigned тип.